### PR TITLE
commit codeclimate config

### PR DESCRIPTION
**Pull request by annaswims**
Commit codeclimate config to .yml so that all who can write to the repo can modify config. 

I took codeclimate defaults + current plugin settings (though I did remove some unnecessary patterns from the `exclude_patterns`)

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+  identical-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+plugins:
+  rubocop:
+    enabled: true
+    channel: rubocop-0-52
+  brakeman:
```
Comment (johnpaulashenfelter) [inline .codeclimate.yml]: Does brakeman have a version/channel? We've got a warning on the newest version when I tried running it locally last week.

File: .codeclimate.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    enabled: true
+  bundler-audit:
+    enabled: true
+exclude_patterns:
+  - 'config/'
```
Comment (johnpaulashenfelter) [inline .codeclimate.yml]: Should we add `docs/`?

File: .codeclimate.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - 'db/'
+  - 'features/'
```
Comment (johnpaulashenfelter) [inline .codeclimate.yml]: This isn't part of our structure

File: .codeclimate.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - 'script/'
```
Comment (johnpaulashenfelter) [inline .codeclimate.yml]: This isn't part of our structure

File: .codeclimate.yml (hunk 1 above)
Comment (annaswims) [reply .codeclimate.yml]: nope. doesn't look like it supports a channel param https://docs.codeclimate.com/docs/brakeman

File: .codeclimate.yml (hunk 3 above)
Comment (annaswims) [reply .codeclimate.yml]: it was part of the defaults, but I'll take it out.  At first I was thinking this WIP was using it https://github.com/department-of-veterans-affairs/vets-api/pull/3102

**Comment by annaswims** [comment]
I intend to come back to the config soon to alter defaults, but this at least gets the config to a state where anyone can change/update it.

**Pull request merged**